pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -641,12 +643,6 @@ stdenv.mkDerivation {
   '';
 
   postInstall = ''
-    # sysinit.target: Don't depend on
-    # systemd-tmpfiles-setup.service. This interferes with NixOps's
-    # send-keys feature (since sshd.service depends indirectly on
-    # sysinit.target).
-    mv $out/lib/systemd/system/sysinit.target.wants/systemd-tmpfiles-setup-dev.service $out/lib/systemd/system/multi-user.target.wants/
```

Comment:
Please put this in an additional PR.

Reply:
Done: https://github.com/NixOS/nixpkgs/pull/171723